bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html

```diff
@@ -18,9 +18,8 @@
 {% endblock %}
 
 {% set fxa_utm_params = {
-  'campaign': 'wnp67',
-  'content': '/firefox/WNP/67/',
-  'source': 'wnp-67'
+  'campaign': 'fxa-embedded-form',
+  'content': 'get-firefox-account'
```

Comment:
In the issue @hoosteeno suggests making utm_content snake_case. I'm not quite clear why that is. Can we just use dashes like here so it's all consistent? (this makes it much easier for devs to remember).

I'm also not quite clear on when utm_content is required, and what does / doesn't make it useful. We know the purpose of this form, so is `get-firefox-account` useful information?

Reply:
I'm not sure what the right answer is here but either way could it please be[ added to the documentation](https://bedrock.readthedocs.io/en/latest/firefox-accounts.html).

Reply:
In regards to content: when I was looking at [this link](https://ga-dev-tools.appspot.com/campaign-url-builder/) shared in [this comment](https://github.com/mozilla/bedrock/pull/7397#issuecomment-509692686) it says that `utm_content` is:
> Used for A/B testing and content-targeted ads. Use utm_content to differentiate ads or links that point to the same URL.
> Examples: logolink or textlink

So perhaps it should only be used in A/B test templates?

Reply:
I take it back, I have an opinion ;) Looking at all our other UTM values I see no underscores so we should stick with dashes unless there is a specific reason for the change here.

Reply:
> we should stick with dashes unless there is a specific reason for the change here

Fine with me, I was just seeking uniformity. If we standardize on kebab case, that's great. The standardization is the main thing.

> I'm also not quite clear on when utm_content is required...
> So perhaps it should only be used in A/B test templates?

This is a good shorthand. All 4 utm parameters are *required* for stub attribution; that fact can confuse us in other contexts like this one, where utm_content is *optional*. As an occasional analyst, I think utm_content can also be useful when we are doing something really different, just to help downstream analysts orient themselves. The "get more from Firefox" claim is pretty different, which is why I thought it might provide some modest value downstream, at minimal cost I think?